docs/content/en/docs/user-guide/plan-preview.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Plan preview"
+linkTitle: "Plan preview"
+weight: 14
+description: >
+  Enables the ability to preview the deployment plan against a given commit before merging.
+---
+
+> NOTE: This feature is currently under Alpha status.
+
+In order to help developers review the pull request with a better experience and more confidence to approve it to trigger the actual deployments,
+PipeCD provides a way to preview the deployment plan of all updated applications by that pull request.
+
+Here are what will be included currently in the result of plan-preview process:
+
+- what application will be deployed once the pull request got merged
+- what deployment strategy (QUICK_SYNC or PIPELINE_SYNC) will be used
+- what resources will be added, deleted, or modified
+
+This feature will available for all application kinds: KUBERNETES, TERRAFORM, CLOUD_RUN, LAMBDA and Amazon ECS.
+
+![](/images/plan-preview-comment.png)
+<p style="text-align: center;">
+PlanPreview with GitHub actions <a href="https://github.com/pipe-cd/actions-plan-preview">pipe-cd/actions-plan-preview</a>
+</p>
+
+## Prerequisites
+
+- Your Pipeds are running at least version `v0.11.0`.
```

Comment:
nit but it's on you
```suggestion
- Your Pipeds are greater than at least `v0.11.0`.
```

Reply:
Let me change this to `Ensure the version of your Piped is at least v0.11.0`.